src/oop/programs_with_multiple_classes/medium/q3/QuestionText.md

```diff
@@ -0,0 +1,19 @@
+# oop-programs_with_multiple_classes-medium-q3
+
+## Java
```

Comment:
We should also format these similarly to how we formatted the questions previously.

Reply:
Like this question for example:

# oop-user_defined_classes-medium-q1

## Question Information:

- Question Type: Java

## Question Text:

In your math class, you have recently been learning about Circles. To model this, you write a Circle class in Java to
help you study. You are given a partially complete Circle class and must complete the rest using the specifications
below. Note that you should not modify the provided portion of the code. Only add the parts specified below to the file.

## Circle Class

The `Circle` class currently has the following items:

- Attributes:
    - An `int` called **position**
    - A `String` called **colour**

You must add the following items to complete the class:

- Constructors:
    - A constructor which is visible from another class and accepts an `int` and a `String` in that order as parameters
      and initializes the **position** and **colour** fields with the input values.